### ISSUE TITLE
The training script is not included in this repository

### ISSUE THREAD

**Issue by lijuntang** [joonson/syncnet_python#18]
Any chance you can share the pytorch version training script now? Thanks

The training script is not included in this repository, but contrastive loss was used to train the model. We used a margin of 20, but using a different value should not affect the performance.

_Originally posted by @joonson in https://github.com/joonson/syncnet_python/issues/7#issuecomment-433331472_
